src/Components/FieldCheckbox/FieldCheckbox.js

```diff
@@ -73,35 +78,38 @@ const defaultProps = {
  */
 
 function FieldCheckbox({
-  id,
-  label,
-  isSelected,
-  isInvalid,
   className,
+  disabled,
   helpText,
   hideLabel,
-  value,
+  id,
+  isInvalid,
+  isSelected,
+  label,
   onChange,
-  disabled,
+  required,
   toggle,
   validationText,
+  value,
   ...rest
 }) {
+  const labelSpan = <span className="label-span">{label}</span>;
```

Comment:
label-span is not descriptive of what it is supposed to be representing. Well technically it is since its a span in a label but class names should not describe markup but rather representation. `label-display` is better (but not best). Its also closer to [BEM style](https://css-tricks.com/bem-101/)

Reply:
That's better. I didn't want to tie it to the required style because that style is only there when it's required.